### DISCUSSION TITLE
Watch Face Maker - Heart Rate Pointer progress problem

### DISCUSSION THREAD

**Discussion by TatouPROD** [zepp-health/zeppos-forum#91] [Q&A] [answered]
Hi,
I want to creat a watch face with an heart rate using a pointer progress.
However, I want to use a specific pointer angle for a specific bpm number (ex: 90° for 60bpm, 180° for 120bpm, etc)
I calculated that the default value for 360° corresponds to approximatly 450bpm. But it's too much for me.
So I tried to do several turns with the pointer but it is not possible.
Can you help me for that ?

**Comment by MyoungXUE** [comment]
Hello, I have already forwarded the problem to the relevant colleagues responsible for the watchface, please wait patiently.

**Comment by zhouxinhe** [comment, marked as answer]
Hello, the heart rate maximum is not fixed, it's difficult to map the pointer progress to a absolute bpm number.
How about to use normal IMG widget (as a pointer) , and get bpm value from hmSensor.id.HEART, calculate the angle and rotate the IMG widget (pointer).

**Reply by TatouPROD** [reply]
Ok thank you.
Is it possible to do that on WatchFaceMaker, I can't find the rotation fuction for an image widget ?

**Reply by idimas13** [reply]
It means the watchface designer should become a developer, then code his own pointer.
Bravo!

**Reply by idimas13** [reply]
What's the problem for develpers to limit heartrate maximum as 220 bpm? It is real biological limit. To make sure, you can use 250.
